src/txmempool.h

```diff
@@ -115,10 +115,10 @@ class CTxMemPoolEntry
     int64_t nSigOpCostWithAncestors;
 
 public:
-    CTxMemPoolEntry(const CTransactionRef& _tx, const CAmount& _nFee,
-                    int64_t _nTime, unsigned int _entryHeight,
-                    bool spendsCoinbase,
-                    int64_t nSigOpsCost, LockPoints lp);
+    CTxMemPoolEntry(const CTransactionRef& tx, const CAmount& fee,
```

Comment:
Feel free to ignore but if you retouch, while changing this function declaration and definition, could also do
```suggestion
    CTxMemPoolEntry(const CTransactionRef& tx, CAmount fee,
```

Reply:
Thanks, done